src/index.js

```diff
@@ -1,5 +1,8 @@
+import keyboardeventKeyPolyfill from 'keyboardevent-key-polyfill'
 import { deprecateComponent } from './utils/deprecate'
 
+keyboardeventKeyPolyfill.polyfill()
```

Comment:
I'm not sure what my resolve on this is yet.  The polyfill does a good job of checking to see if it needs to be applied before running.  At the same time, any consumer who uses stardust is going to have their KeyboardEvent.prototype polyfilled underneath them.  That feels like a bad move.

The util would essentially be exactly what the polyfill is, a match from `keyCode` / `which` values to `key` values.  Just in a function instead of on the proto.

After talking through this, I'm leaning heavily toward util actually.  Will add a TODO to the main PR #206.